lizmap/modules/lizmap/classes/qgisExpressionUtils.class.php

```diff
@@ -171,10 +174,15 @@ public static function virtualFields($layer, $virtualFields, $filter = null, $wi
             'virtuals' => json_encode($virtualFields),
             'with_geometry' => $withGeometry,
             'fields' => $fields,
+            'limit' => $limit,
         );
         if ($filter) {
             $params['filter'] = $filter;
         }
+        if ($sortingField) {
+            $params['sorting_field'] = $sortingField;
+            $params['sorting_order'] = $sortingOrder;
```

Comment:
```suggestion
            $params['sorting_order'] = in_array(strtolower($sortingOrder), array('asc', 'desc')) ? $sortingOrder: 'asc';
```
only available values or throwing an error @mdouchin

Reply:
Ok for me. No need to throw an exception